mantidproject/mantid
# Workbench: Use the selected ws as the default for 1st inputws prop

**Description of work.**
In MantidPlot the selected workspace is automatically the input workspace when an algorithm is executed.

This does the same for workbench

**Report to:** @RichardWaiteSTFC 

**To test:**
1. start workbench
1. create a few workspaces
1. using both the tree view and search box start a few algorithms with different workspaces selected.
1. They should be entered as defaults for the algorithm input workspace property as in mantidplot.


Fixes #27362. 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
